StorageTests/TestLogins.swift

```diff
@@ -87,6 +87,21 @@ class TestSQLiteLogins: XCTestCase {
         let result = logins.getAllLogins().value.successValue!
         XCTAssertEqual(result.count, 2)
     }
+    
+    func testRemoveManyLogins() {
+        log.debug("Remove a large number of logins at once")
+        var guids: [GUID] = []
+        for i in 0..<2000 {
+            let login = Login.createWithHostname("mozilla.org", username: "Fire", password: "fox", formSubmitURL: formSubmitURL)
+            if i <= 1000 {
+                guids += [login.guid]
+            }
+            addLogin(login).value
```

Comment:
It'll take longer on CI. That amount of time makes it worth filing a follow-up `[good-first-bug]` — it's an easy API evolution, and the majority of that time is busy work around starting and committing transactions.